src/helpers.cpp

```diff
@@ -30,7 +31,7 @@ namespace ada::helpers {
       case ada::state::PATH_START: return "Path Start";
       case ada::state::OPAQUE_PATH: return "Opaque Path";
       case ada::state::PORT: return "Port";
-      default: unreachable();
+      default: return "unknown state";
```

Comment:
Is there a specific reason for this?

Reply:
I was trying to silence a warning. I hate static-analysis warnings. They waste our time. But sometimes it is easier to give in.